idaes/models/properties/modular_properties/state_definitions/tests/test_FTPx_electrolyte.py

```diff
@@ -1731,3 +1142,1110 @@ def test_solve_for_true_species(self, frame):
+        for i in m.fs.state[1].flow_mol_phase_comp_apparent:
+            assert (
+                m.fs.state[1].flow_mol_phase_comp_apparent[i]
+                is m.fs.state[1].flow_mol_phase_comp[i]
+            )
+            assert i in m.fs.props.apparent_phase_component_set
+        for i in m.fs.state[1].mole_frac_phase_comp_apparent:
+            assert (
+                m.fs.state[1].mole_frac_phase_comp_apparent[i]
+                is m.fs.state[1].mole_frac_phase_comp[i]
+            )
+            assert i in m.fs.props.apparent_phase_component_set
+
+        # Check for true species components
+        assert isinstance(m.fs.state[1].flow_mol_phase_comp_true, Var)
+        assert len(m.fs.state[1].flow_mol_phase_comp_true) == 8
+        assert isinstance(m.fs.state[1].appr_to_true_species, Constraint)
+        assert len(m.fs.state[1].appr_to_true_species) == 8
+
+    @pytest.mark.component
+    def test_solve_for_true_species(self, frame):
+        m = frame
+
+        m.fs.state[1].flow_mol.fix(2)
+        m.fs.state[1].phase_frac["Liq"].fix(0.5)
+        m.fs.state[1].temperature.fix(300)
+        m.fs.state[1].pressure.fix(1e5)
+
+        m.fs.state[1].mole_frac_comp["H2O"].fix(0.25)
+        m.fs.state[1].mole_frac_comp["CO2"].fix(0.25)
+        m.fs.state[1].mole_frac_comp["KHCO3"].fix(0.25)
+        m.fs.state[1].mole_frac_comp["N2"].fix(0.25)
+
+        m.fs.state[1].mole_frac_phase_comp["Vap", "KHCO3"].fix(1 / 6)
+        m.fs.state[1].mole_frac_phase_comp["Vap", "CO2"].fix(1 / 6)
+
+        assert degrees_of_freedom(m.fs) == 0
+
+        res = solver.solve(m.fs, tee=True)
+
+        # Check for optimal solution
+        assert check_optimal_termination(res)
+
+        # Check true species flowrates
+        assert value(
+            m.fs.state[1].flow_mol_phase_comp_true["Vap", "H2O"]
+        ) == pytest.approx(1 / 6, rel=1e-5)
+        assert value(
+            m.fs.state[1].flow_mol_phase_comp_true["Vap", "CO2"]
+        ) == pytest.approx(1 / 6, rel=1e-5)
+        assert value(
+            m.fs.state[1].flow_mol_phase_comp_true["Vap", "KHCO3"]
+        ) == pytest.approx(1 / 6, rel=1e-5)
+        assert value(
+            m.fs.state[1].flow_mol_phase_comp_true["Vap", "N2"]
+        ) == pytest.approx(0.5, rel=1e-5)
+        assert value(
+            m.fs.state[1].flow_mol_phase_comp_true["Liq", "H2O"]
+        ) == pytest.approx(1 / 3, rel=1e-5)
+        assert value(
+            m.fs.state[1].flow_mol_phase_comp_true["Liq", "CO2"]
+        ) == pytest.approx(1 / 3, rel=1e-5)
+        assert value(
+            m.fs.state[1].flow_mol_phase_comp_true["Liq", "K+"]
+        ) == pytest.approx(1 / 3, rel=1e-5)
+        assert value(
+            m.fs.state[1].flow_mol_phase_comp_true["Liq", "HCO3-"]
+        ) == pytest.approx(1 / 3, rel=1e-5)
+
+
+# -----------------------------------------------------------------------------
+def dens_mol_h2o(*args, **kwargs):
+    return 55e3
```

Comment:
Is this even being called anywhere?

Reply:
It was an artifact from before I extracted the config dictionaries to a separate file.